src/test/java/HomeWork_20.java

```diff
@@ -0,0 +1,66 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork_20 extends BaseTest {
+    @Test
+    public void successfulLoginTest() {
+        login("chernyakma75@gmail.com", "te$t$tudent");
+        WebElement avatar = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("a .avatar")));
+        Assert.assertTrue(avatar.isDisplayed());
+    }
+    @Test
+    public void wrongPasswordLoginTest() {
+        login("chernyakma75@gmail.com", "te$t$tuden");
+        WebElement submitLoginButton = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("button[type='submit']")));
+        Assert.assertTrue(submitLoginButton.isDisplayed());
+    }
+
+    @Test
+    public void emptyPasswordLoginTest() {
+        login("chernyakma75@gmail.com", "");
+        WebElement submitLoginButton = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("button[type='submit']")));
+        Assert.assertTrue(submitLoginButton.isDisplayed());
+    }
+    @Test
+    public void RegistrationNovigation() {
+        WebElement RegistrationButton = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[id='hel']")));
+        RegistrationButton.click();
+        WebElement RegisterButton = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("#button")));
+        Assert.assertTrue(RegisterButton.isDisplayed());
+    }
+    @Test
+    public void playSong() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.